modules/tinymce/src/plugins/table/main/ts/ui/UiUtils.ts

```diff
@@ -17,10 +17,14 @@ interface Item {
 const onSetupToggle = (editor: Editor, formatName: string, formatValue: string) => {
   return (api: Toolbar.ToolbarMenuButtonInstanceApi) => {
     const boundCallback = Singleton.unbindable();
+    const isNone = Strings.isEmpty(formatValue);
 
     const init = () => {
-      api.setActive(editor.formatter.match(formatName, { value: formatValue }));
-      const binding = editor.formatter.formatChanged(formatName, api.setActive);
+      const setActive = (matched) =>
+        api.setActive(isNone ? !matched : matched);
+
+      setActive(editor.formatter.match(formatName, { value: formatValue }, undefined, isNone));
```

Comment:
```suggestion
      // For most checkboxes, we want to make sure that the formatter _does_ match, exactly
      // For the None checkbox, we want to make sure that the formatter _does not_ match, even for similar values
      setActive(editor.formatter.match(formatName, { value: formatValue }, undefined, isNone));
```

Can we add a comment like this (or something else) to explain the logic here? It feels like the sort of thing that's going to be very confusing without the current context